hw_08/Task1/Bot.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task1 {
+    public class Bot {
+        public string Ask(string question) {
+            if (question.Equals("")) {
+                Console.WriteLine("Oops, something went wrong");
+                return "";
+            }
+
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+
+        public bool AskBool(string question) {
+            if (question.Equals("")) {
+                Console.WriteLine("Oops, something went wrong");
+                return false;
+            }
+
+            Console.WriteLine(question + " (да Y, нет N)");
+            string answer = Console.ReadLine();
+            bool bollAnswer = false;
+
+            // можно через приведение к нижнему ригестру
+            switch (answer) {
+                case "да":
+                case "Да":
```

Comment:
in general you can call both `ToLower()` or `ToUpper()` method in `answer` string variable and your switch would look a bit nicer.